content/chapter-02/IndexConfig.md

```diff
@@ -2,17 +2,19 @@
 
 * elasticsearch.yml defines defaults for index configuration:
```

Comment:
I don't think v6 has the default number of shards/replica settings in the elasticsearch.yml